### DISCUSSION TITLE
Got, warn - The utility `w-[calc(100%-theme('spacing[some_key][1.5]'))]` during build

### DISCUSSION THREAD

**Discussion by leonism** [tailwindlabs/tailwindcss#9694] [Help] [answered]
I'm just wondering, each time I'm generating a build, I got the following warning on my console :

```
Rebuilding...

warn - The utility `w-[calc(100%-theme('spacing[some_key][1.5]'))]` contains an invalid theme value and was not generated.

Done in 2787ms.
```

while the thing is, I didn't put any theme value in my `tailwind.config.js`, as per the following

```
/** @type {import('tailwindcss').Config} */
module.exports = {
  content: ["./**/*.{html,js}"],
  theme: {
    extend: {},
  },
  plugins: [],
}

```

And here's how I generated them in the `package.json`, below the "script" line:

`"dev": "npx tailwindcss -i ./assets/css/global.css -o ./assets/css/tailwind.css",`

I'm just wondering, what have I done wrong with my settings, since I only put the default value in my `global.css` 

```
@tailwind base;
@tailwind components;
@tailwind utilities;
```

Any support is highly appreciated, thanks everyone.

Cheers!

**Comment by wongjn** [comment]
As the warning message indicates, `theme('spacing[some_key][1.5]')` is an invalid theme value – perhaps you meant to write `theme('spacing[1.5]')`?

**Comment by leonism** [comment]
I'm a bit puzzled, since in my `tailwind.config.js` file I didn't modify any lines? 

```
/** @type {import('tailwindcss').Config} */
module.exports = {
  content: ["./**/*.{html,js}"],
  theme: {
    extend: {},
  },
  plugins: [],
}
```

as shown, no values provided in the curly bracket after the `theme:`, but then again it might perhaps due to my minor understanding of the setup. 

Thanks,

**Reply by wongjn** [reply]
The error comes from some source code in your project that has the exact string `w-[calc(100%-theme('spacing[some_key][1.5]'))]` – you may have meant to write `w-[calc(100%-theme('spacing[1.5]'))]`. The error is not due to your `tailwind.config.js`.

**Reply by leonism** [reply]
I see, thanks for the swift response.

**Comment by RobinMalfait** [comment, marked as answer]
There are some pieces of code like this in our tests, so if you see them it means that you are watching node_modules files. If you look at your content globs make sure to be explicit e.g.: `./src/**/*.{html,js}` instead of the over generic `./**/*` since this includes the node_modules folder.

**Reply by leonism** [reply]
Thanks Robin, after some minor modification, I solved the issues, I was only doing it for testing purposes.

```
/** @type {import('tailwindcss').Config} */
module.exports = {
  content: ["./*.html"],
  theme: {
    container: {
      center: true,
      padding: "1rem",
    },
    extend: {

    }
  },
  plugins: [],
}
```

I simply edited the line to   `content: ["./*.html"],` and the warning disappeared.
